.github/workflows/python_lint.yml

```diff
@@ -0,0 +1,31 @@
+name: Python Lint Black and flake8
+
+on: [ push, pull_request ]
+
+jobs:
+  lint:
+    name: Lint with Black and flake8
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - uses: actions/setup-python@v1
+      - name: Lint with Black
+        run: |
+          pip install black
+          black --check .
+      - name: If needed, commit black changes to a new pull request
+        if: failure()
+        run: |
+          black .
```

Comment:
Q: Could you add those as blockers. Like check if ```black``` formatting is correct?
```black -v .``` was for this I think.

And the same for pyflake8

Reply:
Ok so only formatting check should be done?

I have added auto formatting action too
which formats itself and pushes a formatted code

Reply:
Yep - put only formatting check - the developer should make sure the code is clean before pushing

Reply:
Added 💯  

@gmuraru 
As you can see here https://github.com/gmuraru/MiceNDemons/pull/25/checks?check_run_id=1219657784
Already there are some files having errors!

And the action will annotate automatically, just add the **`GITHUBTOKEN`** as your repo secret and its done 👍🏻

Reply:
Yep - could you put the limit to 80 characters, please. And after this, the commit should respect a specific format. Even if black/flake8 fails I can merge this and I can put up a fix later for it.